content/docs/forms.md

```diff
@@ -138,7 +138,7 @@ In HTML, `<select>` creates a drop-down list. For example, this HTML creates a d
 </select>
 ```
 
-Note that the Coconut option is initially selected, because of the `selected` attribute. React, instead of using this `selected` attribute, uses a `value` attribute on the root `select` tag. This is more convenient in a controlled component because you only need to update it in one place. For example:
+ध्यान दीजिये की `selected` एट्रिब्यूट की वजह से Coconut ऑप्शन पहले से सेलेक्टेड है। React में, `selected` एट्रिब्यूट इस्तेमाल करने के बजाए, `value` एट्रिब्यूट का इस्तेमाल रुट `select` टैग पर किया जाता है। यह एक कंट्रोल्ड कौम्पोनॅन्ट में बहुत सुविधाजनक रहता है क्यूंकि हमे इसे सिर्फ एक जगह अपडेट करना होता है। उदाहरण के तौर पर:
 
 ```javascript{4,10-12,24}
 class FlavorForm extends React.Component {
```

Comment:
translation inside code and alert messages?

Reply:
It was told that the code need not be translated. Should I do that as well?

Reply:
Comments and messages can be

Reply:
coconut on line 147 got missed